components/Layout.jsx

```diff
@@ -1,8 +1,30 @@
-import { Box, Container } from '@mui/material';
+import { Menu as MenuIcon } from '@mui/icons-material';
+import { Box, Button, Container, SwipeableDrawer } from '@mui/material';
+import { useState } from 'react';
+import Menu from './Menu';
 
 export default function Layout({ children }) {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  function toggleMenu() {
+    setMenuOpen(!menuOpen);
+  }
+
   return (
     <Container maxWidth="sm">
+      {/* TODO: Move Toggle Menu button to a header */}
+      <Button variant="contained" onClick={toggleMenu}>
+        <MenuIcon />
+        Toggle Menu
+      </Button>
```

Comment:
You can use `startIcon` and/or `endIcon`

See: https://mui.com/material-ui/react-button/#buttons-with-icons-and-label